packages/devtools_app/web/index.html

```diff
@@ -10,6 +10,7 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <base href="/">
```

Comment:
@elliette wanted to get your input on changes to this file to verify that this shouldn't have any effects on serving DevTools in g3 or with DDR / the Dart DevTools extension.

Reply:
This looks safe to me! None of the serving logic should care about the base URL.